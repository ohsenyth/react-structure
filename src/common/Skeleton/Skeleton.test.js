import React from 'react';
import { shallow } from 'enzyme';

import Loading from './Skeleton';

describe('Skeleton', () => {
  it('should render correctly', () => {
    const component = shallow(<Loading />);
  
    expect(component).toMatchSnapshot();
  });
});