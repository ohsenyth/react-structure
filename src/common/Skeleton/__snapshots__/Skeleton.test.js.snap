// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Skeleton should render correctly 1`] = `ShallowWrapper {}`;
